ReportSimulationOutput/tests/output_report_test.rb

```diff
@@ -1054,8 +1054,18 @@ def test_timeseries_for_dview
     args_hash = { 'hpxml_path' => File.join(File.dirname(__FILE__), '../../workflow/sample_files/base.xml'),
                   'output_format' => 'csv_dview',
                   'timeseries_frequency' => 'timestep',
-                  'include_timeseries_fuel_consumptions' => true,
-                  'add_timeseries_dst_column' => true }
+                  'include_timeseries_fuel_consumptions' => true }
+    annual_csv, timeseries_csv = _test_measure(args_hash)
+    assert(File.exist?(annual_csv))
+    assert(File.exist?(timeseries_csv))
+    assert_equal('wxDVFileHeaderVer.1', CSV.readlines(timeseries_csv)[0][0].strip)
+  end
+
+  def test_timeseries_for_dview_dst_disabled
+    args_hash = { 'hpxml_path' => File.join(File.dirname(__FILE__), '../../workflow/sample_files/base-simcontrol-daylight-saving-disabled.xml'),
+                  'output_format' => 'csv_dview',
+                  'timeseries_frequency' => 'timestep',
+                  'include_timeseries_fuel_consumptions' => true }
```

Comment:
These tests are pretty unsatisfying. They would pass even if you haven't made any code changes. How about comparing the timeseries_csv rows between base.xml and base-simcontrol-daylight-saving-disabled.xml and asserting that A) the data is identical for the first hour of the year and B) the data is offset by 1 hour in the middle of the year? Or something else that demonstrates the output data is correct.

Reply:
I don't think it's that easy. There are energy use differences between base.xml and base-simcontrol-daylight-saving-disabled.xml. Perhaps we compare something that doesn't change, like `Weather| Drybulb Temperature`?

Reply:
Great points. Yes, using a weather variable should work well.